manifests/backend.pp

```diff
@@ -40,6 +42,7 @@
   Hash $config_hash = {},
   String $url_host = '127.0.0.1',
   Stdlib::Port $url_port = 8080,
+  Boolean $use_ssl = false,
```

Comment:
Do the docs say anything about using a different port when using SSL?

Reply:
The sensu-go docs don't say to use a different port, and it's not necessary.

Reply:
https://docs.sensu.io/sensu-go/5.0/guides/securing-sensu/#securing-the-api-and-the-dashboard